src/main/kotlin/uk/gov/justice/digital/hmpps/bodyscan/service/LimitStatusService.kt

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.digital.hmpps.bodyscan.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.bodyscan.apiclient.BodyScanApiClient
+import uk.gov.justice.digital.hmpps.bodyscan.model.LimitStatusResponse
+import java.time.LocalDate
+import java.time.Year
+
+@Service
+class LimitStatusService(
+  private val prisonApiClient: BodyScanApiClient
+) {
+  companion object {
+    const val TYPE = "BSCAN"
+  }
+
+  fun getLimitStatusForYearAndPrisonNumbers(year: Year, prisonNumbers: List<String>): List<LimitStatusResponse> {
+    val startDate: LocalDate = LocalDate.ofYearDay(year.value, 1)
+    val endDate: LocalDate = LocalDate.ofYearDay(year.value, startDate.lengthOfYear())
+    val apiMap = prisonApiClient.getPersonalCareNeedsForPrisonNumbers(TYPE, startDate, endDate, prisonNumbers)
```

Comment:
meh.. looks like you can do this with adjusters, https://stackoverflow.com/a/33866875